server/dashboards/preconfigured/cosmosdb-handoff.ts

```diff
@@ -143,8 +148,8 @@ export const config: IDashboardConfig = /*return*/ {
           params: {
             databaseId: 'admin',
             collectionId: 'conversations',
-            query: ({ conversationId }) => `SELECT * FROM conversations c 
-              WHERE (c.customer.conversation['$id'] = '${conversationId}')`,
+            query: ({ conversationId }) => `
+              SELECT * FROM conversations c WHERE (c.customer.conversation['$id'] = '${conversationId}')`,
```

Comment:
I meant this:
`
SELECT * FROM conversations c 
WHERE (c.customer.conversation['$id'] = '${conversationId}')`